UPGRADING.md

```diff
@@ -1,5 +1,50 @@
 # Upgrading to 8.0.0
 
+# The wire protocol
+
+The built-in wire protocol has been removed.
+
+The wire protocol is still available by explicitely using the `cucumber-wire` gem.
+
+### Before cucumber 8.0.0
+
+Before cucumber 8.0.0, the wire protocol was automatically installed with cucumber,
+and automatically activated when it had detected a `.wire` file.
+
+If you were using cucumber 7.1.0 and did not already migrate your code, you had a
+deprecation message.
+
+### With cucumber 8.0.0
+
+If you are not using the wire protocol, you have nothing to do.
+
+If you already have updated your code to remove the deprecation message shown when
+using cucumber 7.1.0, you are already up-to-date. Nothing more has to be done.
+
+If you are still using the built-in wire protocol here the step to migrate to cucumber 8.0.0:
+
+- add the gem `cucumber-wire` to your Gemfile alongside the `cucumber` one, and install it:
+  ```ruby
+  # Gemfile
+
+  # ...
+
+  gem "cucumber"
+  gem "cucumber-wire"
+
+  # ...
+
+  ```
```

Comment:
extra triple backticks

Reply:
Looks good to me

There are the one that close the one for the gemfile, then we open a new block for a shell command 🤔

Reply:
![image](https://user-images.githubusercontent.com/24386780/144410446-22f08870-6c8d-4e63-ba26-5d22d5ce7cd2.png)